CHANGELOG.md

```diff
@@ -10,10 +10,17 @@ Every entry has a category for which we use the following visual abbreviations:
 
 ## Unreleased
 
-- 🎁 The JSON import now automatically selects exactly matching types by field
-  name from all known types. The `--type` / `-t` option for `vast import` can
-  now be used as a prefix for selecting matching types. I.e., `-t suricata` will
-  match all types whose name starts with `suricata`.
+- 🎁 VAST now keeps track of the types of imported events and can re-use them
+  for consecutive imports even when they're not specified in the schema. A list
+  of registered types can now be viewed using `vast status` under the key
+  `node.type-registry.types`. [#875](https://github.com/tenzir/vast/pull/875)
```

Comment:
What does this buy me from a user perspective? It's currently written about what happens, not with how the user can benefit from this.

Reply:
> even when they're not specified in the schema

This is the benefit: Once an event of type X was imported successfully, it will always be available regardless of the availability of the original schema.

Reply:
My point is: I'm taking on the user-perspective and cannot interpret the benefit directly. So we need to communicate the message better (-> rewording).